gatsby-config.js

```diff
@@ -1,18 +1,14 @@
 const _ = require('lodash')
-
 const AWS = require('aws-sdk')
 
-require('dotenv').config()
-
 const getSourceS3ConfigForEnvironment = env => {
   switch (env) {
-    case GatsbyEnv.Development: {
-      return {
-        bucketName: 'js-photos-dev',
-        domain: 'minio.jesses.io',
-        protocol: 'https',
-      }
-    }
+    // return {
+    //   bucketName: 'js-photos-dev',
+    //   domain: 'minio.jesses.io',
+    //   protocol: 'https',
+    // }
+    case GatsbyEnv.Development:
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['GatsbyEnv' was used before it was defined. (no-use-before-define)](https://app.codacy.com/app/jessestuart/jesses.io/pullRequest?prid=4576429)